samples/start-app/js-node-express/use-cases/app-activation/notify-authorization-update.js

```diff
@@ -0,0 +1,34 @@
+const { oauth_scopes } = require('./activation');
+
+let doNotifyAuthorizationUpdate = function({
+    httpsClient,
+    tokenDb,
+    LogicError
+ }) {
+    return async function notifyAuthorizationUpdate({req, res, next}) {
+        if (!await tokenDb.hasToken()) {
+            res.render('no_access_token');
+            throw new LogicError('Missing access token in database');
+        } else {
+            const token = await tokenDb.getToken();
+
+            httpsClient.setToken(token);
+
+            const apiUrl = "/connect/apps/v1/scopes/update?scopes=";
+            const options = {
+                path: apiUrl + encodeURIComponent(oauth_scopes.join(' ')),
+                method: 'POST'
+            };
+
+            const response = httpsClient.request(options);
+
+            if (response.hasOwnProperty('code')) {
```

Comment:
response will never have this property if we don't await for it to be filled

Reply:
fixed